go-fuzz-build/main.go

```diff
@@ -605,7 +605,9 @@ func (c *Context) copyFuzzDep() {
 	newDir := filepath.Join(c.workdir, "goroot", "src", "go-fuzz-dep")
 	c.mkdirAll(newDir)
 	dep := c.packageNamed("github.com/dvyukov/go-fuzz/go-fuzz-dep")
-	for _, f := range dep.GoFiles {
+	pkgFiles := dep.GoFiles
```

Comment:
I'm not sure I get the question. `go-fuzz-dep` package is used by the instrumented code and needs to be compiled. So, here we copy both the Go and C/C++ files into the work directory where the `GOPATH` environment is prepared. Since we don't need to instrument this code, we can do this step after we have instrumented the code to be tested

Reply:
I see, I missed that the trampoline was added to `go-fuzz-dep` and thus needs to be picked up here.